compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/TextDecorationLineStyle.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.text
+
+import kotlin.jvm.JvmInline
+
+/**
+ * Defines the style of the text decoration line (e.g. underline).
+ *
+ * The decoration itself is determined via [androidx.compose.ui.text.style.TextDecoration].
+ */
+@ExperimentalTextApi
+@JvmInline
+value class TextDecorationLineStyle internal constructor(val value: Int) {
```

Comment:
I changed the PR description, is it good?

Reply:
> I changed the PR description, is it good?

Maybe write instead "Note that this isn't supported on Android"?

Reply:
Sounds good